wordpress-mobile/gutenberg-mobile
# Add/npm experimental block grid block

This PR is an experiment and an alternative to the https://github.com/wordpress-mobile/gutenberg-mobile/pull/2582. 

It does the following. 
1. It adds the https://github.com/Automattic/block-experiments/pull/136 PR repo into the npm_modules folder. So that it can be accessed by the mobile editor. 
2. It adds a new `npm run start:block-experiments` command that adds a parallel folder called `block-experiments` to metro watch folder list. So that we develop in the `block-experiment` repository that lived parallel to the `mobile-gutenberg` repo. 

To test:
1. Load this PR. Run `npm install`
2. Run `npm run start:reset` notice that you now see the `layout grid` block in the block inserter. You are able to insert the block into the page.
3. Clone the Block Experiment repo in a parallel folder to gutenberg-mobile. 
```
-- gutenberg-mobile
-- block-expriements
```
with  `git clone git@github.com:Automattic/block-experiments.git` 
4. Switch to the `add-mobile-app-support` branch in the `block-experiments` repo. Make a change to the code base so that you will notice that this is the repo being used. 
Such as updating the name of the block on https://github.com/Automattic/block-experiments/blob/master/blocks/layout-grid/src/index.js#L45 
5. Run `npm run start:block-experiments` from the gutenberg-mobile 
6. Notice that the layout grid block now has a new different name.

PR submission checklist:
